GCEWindowsAgent/addresses.go

```diff
@@ -372,3 +379,64 @@ func (a *addressMgr) set() error {
 
 	return nil
 }
+
+var osrelease release
```

Comment:
osRelease

Also, I'm I just missing the place where this gets populated?

Reply:
this var was a placeholder bc this PR isn't based off the PR where i introduced it. will rebase and remove prior to merge.